From 4dc602f79bd6579eef15a9227aee99fe832a7610 Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Fri, 21 Nov 2014 19:36:51 +0100
Subject: dev_manager: fix mknodes

Fix regression introduced with a2c1024f6ac7e22ac5c010a267d0c1df331ee416

_setup_task(mknodes ? name : NULL...

has been replaced with:

_setup_task(type != MKNODES ? name : NULL....

Use '=='

index 44b273d..e336933 100644
--- a/lib/activate/dev_manager.c
+++ b/lib/activate/dev_manager.c
@@ -206,7 +206,7 @@ static int _info_run(info_type_t type, const char *name, const char *dlid,
 			return 0;
 	}
 
-	if (!(dmt = _setup_task(type != MKNODES ? name : NULL, dlid, 0, dmtask,
+	if (!(dmt = _setup_task((type == MKNODES) ? name : NULL, dlid, 0, dmtask,
 				major, minor, with_open_count)))
 		return_0;
 
-- 
cgit v0.10.2

